examples/multiple_data_source_examples/opensearch_dashboards.json

```diff
@@ -0,0 +1,10 @@
+{
+    "id": "multipleDataSourceExample",
```

Comment:
Plugin name is that `multipleDataSourceExample` but I saw somewhere to use `multipleDataSourceExamples`, can we keep name consistent.

Reply:
Good catch! Changed to `multipleDataSourceExamples` in new commit